insecure/corruptible/messageProcessor.go

```diff
@@ -0,0 +1,30 @@
+package corruptible
+
+import (
+	"github.com/rs/zerolog"
+
+	"github.com/onflow/flow-go/network/channels"
+
+	"github.com/onflow/flow-go/model/flow"
+	flownet "github.com/onflow/flow-go/network"
+)
+
+type MessageProcessor struct {
+	logger            zerolog.Logger
+	originalProcessor flownet.MessageProcessor // original message processor
+}
+
+var _ flownet.MessageProcessor = &MessageProcessor{}
```

Comment:
Why keep this if we don't use it? For ingress message processing?

Reply:
This is an engineering best practice in Golang, that we explicitly indicate what interfaces a struct is implementing. We do that through a dummy variable `_` that signals the compiler that it is intentionally of no usage, hence avoiding compile error. In this scenario, we explicitly tell whoever reads the code that this `MessageProcessor` `struct` is supposed to implement the `flownet.MessageProcessor` `interface`. Moreover, if any breaking changes happen that this interface implementation is violated, this line complains of a compile-time error, hence quickly identifying the root cause.